src/ralph/accessories/migrations/0001_initial.py

```diff
@@ -2,17 +2,19 @@
 from __future__ import unicode_literals
```

Comment:
Since this code has already been merged to the default branch, you should not edit this migration. Leave 0001_initial migration as it was and create a new migration with appropriate changes.